cypress/e2e/01-organization_sign_up.cy.js

```diff
@@ -10,7 +10,6 @@ const organization = generateOrganization();
 const user = generateUser();
 
 describe("Organization sign up", () => {
-  loginHooks();
```

Comment:
since we're refactoring this to use the actual `cy.session` wrapper, they need to be moved into a proper testing hook rather than being called outside like it is here.